test/zdtm/static/timens_for_kids.c

```diff
@@ -9,7 +9,7 @@
 
 #include "zdtmtst.h"
 
-const char *test_doc = "Check nested time namespaces";
+const char *test_doc = "Check ability to create nested time namespaces";
```

Comment:
This isn't quite right. We check whether criu handles a specific configuration properly.

Reply:
I am still very new to criu project so pardon me if this is a really obvious question. Can you please expand on "check whether criu handles a specific configuration properly", what kind of behavior is expected from criu?

Reply:
@VermaSh perhaps "Check ability to _restore_ nested time namespaces" might be more appropriate description of this test

Reply:
Sorry, got distracted by some other work. But thank you for the suggestions @rst0git, I'll update the PR.